### ISSUE TITLE
Using <i> tag for icons breaks A11Y

### ISSUE THREAD

**Issue by smartm0use** [fomantic/Fomantic-UI#754]
Do you know that using `<i>` tag for showing icons is not compliant with A11Y?
We should use something like this:

`<span class="icon cancel"></span>`

instead of:

`<i class="icon cancel"></i>`

**Comment by lubber-de** [comment]
I suggest the approach font awesome described here https://origin.fontawesome.com/how-to-use/on-the-web/other-topics/accessibility and add aria labels in your application according to your needs. 

However, we may take a look at the Fui dropdown and search templates where icons are generated to add aria properties 

Changing from I to span is not an option for v2. It will break tons application's dramatically

**Comment by smartm0use** [comment]
The problem is with the tag `i` that should stand for other, according to [W3C](https://www.w3schools.com/tags/tag_i.asp).

Try to add the following code in an accessibility checker (like [this](https://achecker.ca/checker/index.php)):

```
<!DOCTYPE html>
<html lang="en"> 
    <head>
	<title>A11Y icons test</title>
    </head>

    <body>
        <div class="home">
            <i class="icon cancel"></i>Icon test
        </div>
    </body>
</html>
```

You will get this error:

**Check 117: i (italic) element used.
Repair: Replace your i elements with em or strong.**

**Comment by hammy2899** [comment]
This can't be fixed in v2 without breaking the icon component. The only thing we can do is add the span element to the CSS selectors as well as the i tag.

**Comment by smartm0use** [comment]
On my forked Fui I have replaced all occurrences of tag `i` with tag `span` that has `icon-fix` class.

**Comment by lubber-de** [comment]
> The only thing we can do is add the span element to the CSS selectors as well as the i tag.

This would add a ton of extra CSS declarations. Maybe we can adjust the https://github.com/fomantic/create-fomantic-icons script using a variable like `iconTag: 'i'`, so the user can compile with `iconTag: 'span'` if needed? 
But this would also mean to adjust the `icon.less` accordingly 🤔

**Comment by hammy2899** [comment]
@lubber-de Yeah, IMO its too large of a change for now and we should address this when we do the icon changes in v3.

**Comment by hbriggs** [comment]
Do you think this is why FA icons rendered with `i` tags do not render on Safari?  👈 and is this a known issue (I was searching for an open issue but haven't found one yet...)?
![Screen Shot 2019-05-20 at 11 25 12 AM](https://user-images.githubusercontent.com/4731354/58043451-0e2cc480-7af2-11e9-86c8-dbd8dff406f4.png)
?

**Comment by hbriggs** [comment]
re: my question, it looks like it could be caused by [this bug](https://github.com/FortAwesome/Font-Awesome/issues/12339) with font awesome so probably nothing to do here...

thanks for all the awesome work! so glad someone picked up the semantic UI project!! 👏 👏 👏

**Comment by smartm0use** [comment]
> @lubber-de Yeah, IMO its too large of a change for now and we should address this when we do the icon changes in v3.

I have already done this work. If you need I can create a PR.

**Comment by mlopezgva** [comment]
In my opinion, A11Y test does not take into acount that:

- `<i>` tag is not being used to emphasize any text, because
- that tag is *empty* so there shouldn't be _anything_ to complain about.

It is important *when using icons only* (`ui icon menu` or a `ui icon buttons` without text, for example), and, again, there is *no text*, so an `aria-` attribute would be enforced to inform about the element's meaning.

I am not versed in this A11Y thing, but complaining about incorrectly emphasize an *empty* node seems outdated to me: using `<i>` for *i*con has been done for almost a decade by many frameworks, and the authority behind the A11Y standardization should be noticed and the standard updated.
